docs/source/workshop.rst

```diff
@@ -1,4 +1,361 @@
-Workshop Template
-===================
+Creating a conference
+========================
+
+Demo Workshop
+---------------
+
+This is a demo script. This script captures creation of a new conference on OpenReview system.
+
+Some of the tasks described below (such as creation of a conference sub-group, creation of a submission, etc) can only be done using an administrator profile.
+
+If you are registered as an administrator with OpenReview, you should already have admin privileges on a group representing your conference (e.g. ICML.cc/2019/Conference).
+
+Following are the steps and brief explanations.
+
+
+Login
+---------
+
+Login can be done through the API like this::
+
+    >>> import openreview
+	>>> client = openreview.Client(baseurl='https://openreview.net',username=<your username>,password=<your password>)
+
+Also, some of the Openreview services can be accesses by a guest (without loggin in). This can be done through API like this::
+
+    >>> guest_client = openreview.Client(baseurl='https://openreview.net')
```

Comment:
how to login as a guest client could be placed in the installation and setup section.